docs/UserGuide.md

```diff
@@ -346,12 +397,10 @@ Examples:
 
 Finds students whose `TAG`s contain any of the given keywords.  
 Note that you have to run `list` to display all the students again.
-Note `Students` list in the `classes` tab will only show students found, `size` of the class will remain unchanged 
-despite filtering.
 
 Format:
 ```
-findtag KEYWORD, [MORE_KEYWORDS]
+findtag TAG, [MORE_TAGS]
```

Comment:
I think 'keyword' might be fine here, because the user might be searching for only part of a tag, like 'math' for all the 'A Math' and "C Math" tags

Reply:
Good point, I'll define what a 'keyword' refers to then haha